### PR TITLE
BUG-418589 - fixed README

### PR DESCRIPTION
**Pull request by JustynaApryasz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Vehicle Damage Modeling
 
-This repository contains template projects for Vehicle Damage Modeling project.
+This repository contains Vehicle Damage Modeling project.
 
 ## iOS
 
+iOS project uses SceneKit - a 3D graphics application programming interface (API) for Apple Inc.
```
Comment (ghawliczek) [inline README.md]: for Apple Inc. -> designed by Apple Inc.

Comment (ghawliczek) [inline README.md]: for Apple Inc. -> developed by Apple Inc.

File: README.md [hunk 2]
```diff
@@ -13,11 +15,8 @@ This repository contains template projects for Vehicle Damage Modeling project.
 ### Structure
 There are two projects:
 
- - _VehicleDamageModeling_ - template for developing framework with UI component for choosing damaged parts of vehicle
- - _SampleApp_ - sample application which demonstrates how to import and use _VehicleDamageModeling_ framework
-
-### Notice
-iOS project uses [Metal Framework](https://developer.apple.com/metal/) for rendering. As some of required classes (like `CAMetalLayer`) is not available on iOS Simulator - please use iOS Device as compilation target.
+ - _FVM_ - developing framework with UI component for choosing damaged parts of vehicle
```
Comment (ghawliczek) [inline README.md]: developing framework with UI component for choosing -> framework containing UI component that allows user to choose

File: README.md [hunk 3]
```diff
@@ -13,11 +15,8 @@ This repository contains template projects for Vehicle Damage Modeling project.
...
+ - _SampleApp_ - sample application which demonstrates how to import and use _FVM_ framework
```
Comment (SzymonSmykala) [inline README.md]: why not just "demonstrates"? You can also use framework, not only import :)

File: README.md [hunk 4]
```diff
@@ -1,9 +1,11 @@
...
-This repository contains template projects for Vehicle Damage Modeling project.
+This repository contains Vehicle Damage Modeling project.
 
 ## iOS
 
+iOS project uses SceneKit - a 3D graphics application programming interface (API) developed by Apple Inc. 
+
 ### Requirements
 
  - XCode 10.1
```
Comment (lapwingg) [inline README.md]: Please change target SDK to 9

File: README.md [hunk 5]
```diff
@@ -1,9 +1,11 @@
 # Vehicle Damage Modeling
 
-This repository contains template projects for Vehicle Damage Modeling project.
+This repository contains Vehicle Damage Modeling project.
 
 ## iOS
 
+iOS project uses SceneKit - a 3D graphics application programming interface (API) developed by Apple Inc.
```
Comment (lapwingg) [inline README.md]: Please highlight 'SceneKit' the same as below is highlighted libGdX

File: README.md (hunk 3 above)
Comment (lapwingg) [reply README.md]: Maybe you should change to: 
"sample application which demonstrates usage of _FVM_ framework "

File: README.md (hunk 5 above)
Comment (ghawliczek) [reply README.md]: +1

**Pull request closed without merging**